src/test/java/com/puppycrawl/tools/checkstyle/internal/XpathRegressionTest.java

```diff
@@ -203,13 +205,19 @@ protected String getResourceLocation() {
 
     @Test
     public void validateIncompatibleJavadocCheckNames() throws IOException {
-        final Set<Class<?>> abstractJavadocCheckNames = CheckUtil.getCheckstyleChecks()
+        // subclasses of AbstractJavadocCheck
+        final Stream<Class<?>> abstractJavadocCheckNames = CheckUtil.getCheckstyleChecks()
                 .stream()
-                .filter(AbstractJavadocCheck.class::isAssignableFrom)
+                .filter(AbstractJavadocCheck.class::isAssignableFrom);
+        // hardcoded exceptions, will eventually include JavadocMethodCheck and JavadocTypeCheck
+        final Stream<Class<?>> otherJavadocCheckNames = Stream.of(JavadocStyleCheck.class);
```

Comment:
It is possible we will have other checks join this list. Please move it to a field.

Reply:
Ok, I'll move it out as a constant next to the definition of `INCOMPATIBLE_JAVADOC_CHECK_NAMES`